models/issue_list.go

```diff
@@ -441,31 +488,31 @@ func (issues IssueList) loadTotalTrackedTimes(e Engine) (err error) {
 // loadAttributes loads all attributes, expect for attachments and comments
 func (issues IssueList) loadAttributes(e Engine) (err error) {
```

Comment:
Probably don't need a named err here any more.

Reply:
Right, fixed it.